fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java

```diff
@@ -1894,6 +1895,20 @@ private void enableLightSchemaChange(Database db, OlapTable olapTable) throws Dd
         alterLSCHelper.enableLightSchemaChange();
     }
 
+    public void replayAlterLSC(AlterLSCOperationLog info) throws MetaNotFoundException {
```

Comment:
`replayAlterLSC` -> `replayAlterLightSchemaChange`,   the abbreviation `LSC ` is not CamelCase